DualView/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
```

Comment:
I think this is a better `.gitignore` file: https://github.com/microsoft/surface-duo-compose-samples/blob/main/ExtendedCanvas/.gitignore